make/hotspot/lib/CompileJvm.gmk

```diff
@@ -88,6 +88,12 @@ DISABLED_WARNINGS_gcc := array-bounds comment delete-non-virtual-dtor \
     maybe-uninitialized missing-field-initializers parentheses \
     shift-negative-value unknown-pragmas
 
+ifeq ($(DEBUG_LEVEL), fastdebug)
+  ifeq ($(call And, $(call isTargetOs, linux) $(call isTargetCpu, aarch64)), true)
```

Comment:
The idiomatic way we have expressed this in other places in the JDK build is:
```suggestion
  ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, aarch64), true+true)
```

Reply:
I don't have a strong preference, but the `And` macro was actually added by you in https://bugs.openjdk.org/browse/JDK-8218431 which is where the new isTarget* macros were added, so the intention then was to use it for exactly this purpose.

Reply:
FWIW I just followed the preexisting pattern in the same file, e.g. https://github.com/openjdk/jdk/blob/7d25f1c6cb770e21cfad8096c1637a24e65fab8c/make/hotspot/lib/CompileJvm.gmk#L1100. @magicus Let me know if you want me to change it.

Reply:
> [T]he And macro was actually added by you

Oh. 😊 Ohh... That was a bit stupid of me, then. :) But I just checked, I found 4 cases of `$(call And)` vs 11 cases of using `+` in our makefiles. So it is a clear overweight for the `+` syntax, but is not the sole pattern as I thought.

So yes, it is okay to keep it as it is.

Reply:
Leaving it as-is then. Thank you.